migrations/add-plugin-to-user-ident.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "user" ADD COLUMN plugin varchar;
+UPDATE "user" SET plugin = 'learn';
+ALTER TABLE "user" ALTER COLUMN plugin SET NOT NULL;
+ALTER TABLE "user" DROP CONSTRAINT unique_user;
+ALTER TABLE "user" ADD CONSTRAINT "unique_user" UNIQUE(ident, plugin);
```

Comment:
It looks like it does drop and create it. I'm still getting used to the idea of auto-migrating some fields.